fusillade/api/groups/__init__.py

```diff
@@ -1,14 +1,15 @@
 from flask import request, make_response, jsonify
 
 from fusillade import directory, Group
-from fusillade.utils.authorize import authorize
 from fusillade.api.paging import get_next_token, get_page
+from fusillade.utils.authorize import authorize
 
 
 @authorize(['fus:PostGroup'], ['arn:hca:fus:*:*:group'])
 def post_group(token_info: dict):
     json_body = request.json
-    group = Group.create(directory, json_body['group_id'], statement=json_body.get('policy'))
+    group = Group.create(directory, json_body['group_id'], statement=json_body.get('policy'),
+                         creator=token_info['https://auth.data.humancellatlas.org/email'])
     group.add_roles(json_body.get('roles', []))  # Determine what response to return if roles don't exist
     return make_response(f"New role {json_body['group_id']} created.", 201)
```

Comment:
Does this always return 201 what if something  in group.create() fails?

Reply:
If the object already exists a 409 is returned. There are a few other errors that can be returned if this failed. It would probably be a good idea if I moved all HTTP error into the API side of the code. Currently it's in the clouddirectory.py code.

Reply:
The benefit of  being in clouddirectory is there is less repeated code. Moving these error to fusillade/api/... would add more code duplication.